i18n/GLOSSARY_de.md

```diff
@@ -20,14 +20,19 @@ Dieses Glossar begleitet die [GitOps-Prinzipien](./PRINCIPLES_de.md) und andere
 
     Abweichung bezeichnet eine (beginnende) Entfernung des Ist-Zustandes eines Systems vom gewünschten [Soll-Zustand](#soll-zustand).
 
+- ## Beziehen der Änderungen (Pull)
+
+
+    [Prinzip 3](./PRINCIPLES_de.md) (Automatisch bezogen) spezifiziert, dass der gewünschte Zustand "gezogen" (Pull) statt von außen "gedrückt" (Push), werden muss, hauptsächlich weil die Software-Agenten in der Lage sein müssen, den gewünschten Zustand aus dem Zustandsspeicher zu jeder Zeit abzurufen – nicht nur dann, wenn es eine absichtliche Änderung im Zustandsspeicher gibt, die ein Push-Ereignis auslöst.
```

Comment:
> (Automatisch bezogen)

I'm wondering if we should change this to the english term (Pulled Automatically) to find the relation to the "technical terms" easier. This would be a more general thing to discuss in the project
What do you think @scottrigby, @christianh814 - would this be a topic for one of the meetings?

Reply:
We talked about adding the english words next to the German word. For example `(Automatisch bezogen (eng. Pulled Automatically)). What do you think about adding those in the Glossary and the Principles doc @mstein11 ?